readers/common/src/main/java/com/criteo/hadoop/garmadon/reader/metrics/PrometheusHttpConsumerMetrics.java

```diff
@@ -0,0 +1,87 @@
+package com.criteo.hadoop.garmadon.reader.metrics;
+
+import com.criteo.hadoop.garmadon.reader.GarmadonReader;
+import io.prometheus.client.Counter;
+import io.prometheus.client.Gauge;
+import io.prometheus.client.exporter.HTTPServer;
+import io.prometheus.client.hotspot.DefaultExports;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.management.*;
+import java.io.IOException;
+import java.lang.management.ManagementFactory;
+import java.util.Optional;
+import java.util.concurrent.ScheduledThreadPoolExecutor;
+import java.util.concurrent.TimeUnit;
+
+/**
+ */
+public class PrometheusHttpConsumerMetrics implements ConsumerMetrics {
+    private static final Logger LOGGER = LoggerFactory.getLogger(PrometheusHttpConsumerMetrics.class);
+    private static final MBeanServer MBS = ManagementFactory.getPlatformMBeanServer();
+
+    public static String RELEASE = Optional
+            .ofNullable(PrometheusHttpConsumerMetrics.class.getPackage().getImplementationVersion()).orElse("1.0-SNAPSHOT");
+
+    public static final Counter garmadonReaderMetrics = Counter.build()
+            .name("garmadon_reader_metrics").help("Garmadon reader metrics")
+            .labelNames("name", "hostname", "release")
+            .register();
+
+    private static final Gauge baseKafkaMetricsGauge = Gauge.build()
+            .name("garmadon_kafka_metrics").help("Kafka producer metrics")
+            .labelNames("name", "hostname", "release")
+            .register();
+
+    public HTTPServer server;
+
+    private static ScheduledThreadPoolExecutor scheduler = new ScheduledThreadPoolExecutor(1);
+
+    private static ObjectName oname;
+
+    static {
+        // Expose JMX, GCs, classloading, thread count, memory pool
+        DefaultExports.initialize();
+        try {
+            oname = new ObjectName("kafka.consumer:type=consumer-metrics,client-id=" + GarmadonReader.CONSUMER_PREFIX_NAME + "." + GarmadonReader.HOSTNAME);
```

Comment:
GarmadonReader.CONSUMER_PREFIX_NAME + "." + GarmadonReader.HOSTNAME -> code already used somewhere else. I propose to dedicate a property for it